### DISCUSSION TITLE
Is it possible to completely disable gossip?

### DISCUSSION THREAD

**Discussion by seungyongshim** [AsynkronIT/protoactor-dotnet#1964] [Q&A]
Excuse me,

Is it possible to completely disable gossip in a Proto.Actor cluster deployed on EKS? I am not familiar with the functionality of gossip, but I understand that it is responsible for health checks on each node. Is it possible to bypass this feature using EKS capabilities?

Thank you.

**Comment by rogeralsing** [comment]
To disable gossip member expiration:
```
ClusterConfig.WithHeartbeatExpiration(TimeSpan.Zero);
```

There is however no way to completely disable gossip, as it is used for other features also..

**Comment by seungyongshim** [comment]
I would like to completely remove the gossip feature because it causes unreproducible errors due to its impact on the infrastructure. I am wondering if it is possible to resolve the affected features through EKS if we remove gossip. I am curious if it is possible to configure the minimum features required to set up the cluster without gossip.

Of course, I understand that certain functionalities such as PubSub in the cluster cannot be achieved without gossip.

Best regards

**Reply by rogeralsing** [reply]
Can you describe the issue you are seeing?
If you are seeing gossip timeouts, that is likely an indication of an under-provisioned system that cannot operate fast enough.
the messages sent by gossip is no different from other messages, so if they timeout, so will "normal" messages

In K8s, it's usually an issue with resource limits on CPU

**Reply by seungyongshim** [reply]
> Can you describe the issue you are seeing? If you are seeing gossip timeouts, that is likely an indication of an under-provisioned system that cannot operate fast enough. the messages sent by gossip is no different from other messages, so if they timeout, so will "normal" messages
> 
> In K8s, it's usually an issue with resource limits on CPU

I knew. but we prioritize resource limits over gossip.
